presto-accumulo/src/test/java/com/facebook/presto/accumulo/model/TestRow.java

```diff
@@ -108,7 +108,7 @@ public void testRowFromString()
         schema.addColumn("m", Optional.of("m"), Optional.of("m"), VARCHAR);
         schema.addColumn("n", Optional.of("n"), Optional.of("n"), VARCHAR);
 
-        Row actual = Row.fromString(schema, "a,b,c|true|1999-01-01|123.45678|123.45678|12345678|12345678|12345|12:30:00|1999-01-01 12:30:00.0|123|O'Leary|O'Leary|", '|');
+        Row actual = Row.fromString(schema, "a,b,c|true|1999-01-01|123.45678|123.45678|12345678|12345678|12345|12:30:00|1999-01-01 12:30:00.000|123|O'Leary|O'Leary|", '|');
```

Comment:
This was done to match the TIME_STAMP format that we have defined in `TIMESTAMP_PARSER`
`yyyy-MM-dd HH:mm:ss.SSS`
Joda library seemed to be a bit forgiving with using both `.S` and `.SS` when `.SSS` is expected